cumulus/parachains/runtimes/bridge-hubs/common/src/message_queue.rs

```diff
@@ -23,7 +23,7 @@ use frame_support::{
 use pallet_message_queue::OnQueueChanged;
 use scale_info::TypeInfo;
 use snowbridge_core::ChannelId;
-use xcm::v4::{Junction, Location};
+use xcm::v5::{Junction, Location};
```

Comment:
Would it be better to use `prelude` or `latest` here?

Reply:
@yrong yes, sure, please, you can open PR directly to the master with fixing message_queue.rs to use latest - it will help reducing XCMv5 PR size.

and to your minor fix, it is not that correct, because pools are using `xcm::v4::Location`, we need possibly migration for that, so using there `xcm:v4` there is correct